plugin/src/main/java/org/wildfly/plugin/provision/PackageServerMojo.java

```diff
@@ -492,6 +564,9 @@ private void warnExtraConfig(Path extraContentDir) {
 
     protected Path getDeploymentContent() throws MojoExecutionException {
         final PackageType packageType = PackageType.resolve(project);
+        if (packageType.getPackaging().equals("pom")) {
```

Comment:
This should probably use `packageType.isIgnored()`.

Reply:
Yes!

Reply:
This doesn't appear to be changed.